connector/datadogconnector/config.go

```diff
@@ -75,6 +76,11 @@ type TracesConfig struct {
 
 	// ResourceAttributesAsContainerTags specifies the list of resource attributes to be used as container tags.
 	ResourceAttributesAsContainerTags []string `mapstructure:"resource_attributes_as_container_tags"`
+
+	// BucketInterval specifies the time interval size of aggregation buckets that aggregate the Datadog trace metrics.
+	// It is also the time interval that Datadog trace metrics payloads are flushed to the pipeline.
+	// Default is 10s if unset.
```

Comment:
where is the default set ? i don't see it in the PR

Reply:
Default is set by trace agent config: https://github.com/DataDog/datadog-agent/blob/7f37c8d76776d7a5267946683e6931a3a177f68a/pkg/trace/config/config.go#L477